benchmarks/hash_table/static_set_perfect/contains_bench.cu

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Is it possible to use the existing file `../static_set/contains_bench.cu` and then template it so that it receives the probing scheme as a parameter?  And then add to `NVBENCH_TYPE_AXES` the probing scheme?

I don't know anything about how nvbench works.  Maybe it's not possible?  Or maybe the output would look too wonky?